integration-tests/__tests__/location_in_results.test.js

```diff
@@ -28,6 +28,6 @@ it('adds correct location info when provided with flag', () => {
   const assertions = result.testResults[0].assertionResults;
   expect(result.success).toBe(true);
   expect(result.numTotalTests).toBe(2);
-  expect(assertions[0].location).toEqual({column: 1, line: 9});
-  expect(assertions[1].location).toEqual({column: 3, line: 14});
+  expect(assertions[0].location).toEqual({column: 1, line: 10});
+  expect(assertions[1].location).toEqual({column: 2, line: 15});
```

Comment:
@thymikee your comment disappeared, but here's my response:

I'm a bit unsure about the column change but line wise it was correct earlier as well up until I messed up the `'use strict';`, which meant that babel changes the file. Then it was wrong and pointed to the wrong lines.

`location` without passing it through source map:

```js
{"column": 1, "line": 12}
{"column": 3, "line": 17}
```

Reply:
Hm, looking at the source:

<img width="288" alt="screen shot 2018-02-18 at 09 22 54" src="https://user-images.githubusercontent.com/5106466/36349846-56611db8-148d-11e8-8e69-673435a5ee4d.png">


shouldn't this be:

```js
{"column": 3, "line": 11}
{"column": 5, "line": 16}
```
?

Reply:
No, it's the `it`s, not the `expect`s

Reply:
Ah pardon me. Then te second column should be 3 instead of 2.

Reply:
Yup, it should (I just tested with normal js - it's not zero-indexed)

Reply:
This is a bug in either `babel` or `source-map`.

See this example:

```js
'use strict';

const fs = require('fs');
const {transform} = require('babel-core');
const {SourceMapConsumer} = require('source-map');

const fileContent = fs.readFileSync(
  './integration-tests/location-in-results/__tests__/test.js',
  'utf8',
);

const res = transform(fileContent, {
  plugins: ['transform-strict-mode'],
  sourceMaps: 'both',
});

const consumer = new SourceMapConsumer(JSON.stringify(res.map));

console.log(
  // 17:3 in `res.code` has the `it('also works', () => {`
  consumer.originalPositionFor({
    column: 3,
    line: 17,
  }),
);
```

Reply:
Created https://github.com/SimenB/sourcemap-issues with the reproduction.

@jwbay any ideas here? Should we go to babel or source-map and report it? Or am I doing something really weird?